src/src/WebServer/CacheControllerPages.cpp

```diff
@@ -23,76 +26,197 @@
 void handle_dumpcache() {
   if (!isLoggedIn()) { return; }
 
-  C016_startCSVdump();
-  unsigned long timestamp;
-  uint8_t  pluginID;
-  uint8_t  TaskIndex;
-  Sensor_VType  sensorType;
-  uint8_t  valueCount;
-  float val1;
-  float val2;
-  float val3;
-  float val4;
+  // Filters/export settings
+  char separator     = ';';
+  bool joinTimestamp = false;
+  bool onlySetTasks  = false;
 
-  TXBuffer.startStream();
-  addHtml(F("UNIX timestamp;contr. idx;sensortype;taskindex;value count"));
 
-  for (taskIndex_t i = 0; i < TASKS_MAX; ++i) {
-    for (int j = 0; j < VARS_PER_TASK; ++j) {
-      addHtml(';');
-      addHtml(getTaskDeviceName(i));
-      addHtml('#');
-      addHtml(getTaskValueName(i, j));
+  if (hasArg(F("separator"))) {
+    String sep = webArg(F("separator"));
+
+    if (isWrappedWithQuotes(sep)) {
+      removeChar(sep, sep[0]);
     }
+    separator = sep[0];
+  }
+
+  if (hasArg(F("jointimestamp"))) {
+    joinTimestamp = true;
   }
-  html_BR();
-  float csv_values[VARS_PER_TASK * TASKS_MAX];
 
-  for (int i = 0; i < VARS_PER_TASK * TASKS_MAX; ++i) {
-    csv_values[i] = 0.0f;
+  if (hasArg(F("jointimestamp"))) {
+    joinTimestamp = true;
+  }
```

Comment:
This seems to be duplicated? Or has GH messed something up?

Reply:
Unsaved file before commit....
But thanks for looking into the code anyway :)